src/main/java/net/jodah/failsafe/SyncFailsafe.java

```diff
@@ -114,6 +115,27 @@ public void run(ContextualRunnable runnable) {
   }
 
   /**
+   * Creates a proxy instance of the provided {@link T} instance such that all public methods
+   * use the {@link RetryPolicy} and {@link CircuitBreaker} set.
+   * The returned proxy adheres to the interface provided except for two cases:
+   * <ol>
+   *     <li>CircuitBreakerException: The methods may throw this type of
+   *     exception if the circuit breaker is open.</li>
+   *     <li>UndeclaredThrowableException: If you provide a fallback which throws
+   *     an exception undeclared by the interface.</li>
+   * </ol>
+   * @param instance the instance to wrap a proxy around
+   * @param clazz the type of the parameter needed due to erasure. This should be an interface.
```

Comment:
For now I think it should say: _This must be an interface type._ (Same as the next line.)